docs/plans/2026-05-31_h3-v2-ga-upgrade.md

```
@@ -0,0 +1,13 @@
+# h3 v2 GA Upgrade Follow-up
+
+## Summary
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)